### ISSUE TITLE
Redmi 8A SIM 1 and SIM 2 swapped in About phone

### ISSUE THREAD

**Issue by milankragujevic** [phhusson/treble_experimentations#1053]
In About phone, in the first screenshot, the SIMs are arranged correctly (SIM 1 is Telenor and it is actually in SIM 1 slot of the device according to the markings on the SIM tray, and Vip SRB is SIM 2).

When tapping on either of the SIMs, it opens a modal where the other SIM is displayed. So, tapping on SIM 1 will show "SIM status (sim slot 1)" but show data for Vip SRB (which is actually SIM 2). Tapping on SIM 2 on that screen will show "SIM status (sim slot 2)" but show data for Telenor which is SIM 1 in reality.

Same for the notifications bar, the first signal icon has an ×, indicating Data is disabled, and the second signal icon has no ×, indicating data is enabled. However, SIM 1 has data enabled, and SIM 2 does not (Data SIM is set to SIM 1 - Telenor, and the phone really uses that SIM card for data).

Going into Mobile network settings, first SIM is shown as UNKNOWN but under Network it shows Telenor (which is correct) and Data is enabled. Second SIM is shown as VIP SRB, which is correct, however under network it shows Telenor (which is not correct, and Auto select network is enabled for both SIMs). Data is disabled, as it is in reality.

I am sure that Vip SRB has no national roaming on Telenor network which could cause such issues. Roaming is turned off for both SIMs.

Basically, this is just a display bug, as the modem functions normally. Sending and receiving SMS is shown properly (Telenor as SIM1, Vip SRB as SIM2), as well as phone calls, USSD codes...

*#*#4636#*#* shows only SIM 1 as Telenor. 

![Screenshot_20200105-225803](https://user-images.githubusercontent.com/7611163/71786719-fa958600-300e-11ea-9b83-82f5be04fd25.png)
![Screenshot_20200105-225815](https://user-images.githubusercontent.com/7611163/71786720-fc5f4980-300e-11ea-91d1-cd21cc87b7ed.png)
![Screenshot_20200105-225809](https://user-images.githubusercontent.com/7611163/71786721-fd907680-300e-11ea-8914-e450a6142d74.png)
